deployment/scripts/install.sh

```diff
@@ -183,27 +190,32 @@ function generate_templates() {
   oc delete template --selector logging-infra=fluentd
   oc delete template --selector logging-infra=elasticsearch
 
+  image_params=$(join , \
+            IMAGE_VERSION_DEFAULT=${image_version} \
+            IMAGE_PREFIX_DEFAULT=${image_prefix} \
+            )
+
   create_template_optional_nodeselector "${ES_NODESELECTOR}" es \
     --param ES_CLUSTER_NAME=es \
     --param ES_INSTANCE_RAM=${es_instance_ram} \
     --param ES_NODE_QUORUM=${es_node_quorum} \
     --param ES_RECOVER_AFTER_NODES=${es_recover_after_nodes} \
     --param ES_RECOVER_EXPECTED_NODES=${es_recover_expected_nodes} \
     --param ES_RECOVER_AFTER_TIME=${es_recover_after_time} \
-    --param IMAGE_VERSION_DEFAULT=${image_version}
+    --param "$image_params"
```

Comment:
well it's not strictly necessary; the trigger for getting the tag from the imagestream is supposed to overwrite whatever we put in the image field. however if that doesn't happen for some reason, and the user manually deploys, it's really confusing if the image is not what we intended.